services/views/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "pins-views",
+  "version": "1.0.0",
+  "description": "nunjucks views used by pins",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
+    "format": "prettier --write './**/*.js'",
+    "test": "jest --coverage --maxWorkers=4"
+  },
+  "dependencies": {
+    "govuk-frontend": "^3.13.0",
+    "pino": "^6.7.0"
+  },
+  "devDependencies": {
+    "eslint": "^7.11.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.14.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "lint-staged": "^10.5.0",
+    "patch-package": "^6.2.2",
+    "pino-pretty": "^4.3.0",
```

Comment:
It doesn't look like we're doing any logging so do we need pino-pretty as a dependency?

Reply:
Removed